text_summarization/Properties/serviceDependencies.json

```diff
@@ -6,9 +6,10 @@
     "secrets1": {
```

Comment:
Does this file need to be committed? Not challenging it, just never clear on if these files need to be committed or if committing them is discouraged.